services/sensorhub-service-video/build.gradle

```diff
@@ -7,6 +7,13 @@ dependencies {
   implementation 'org.sensorhub:sensorhub-core:' + oshCoreVersion
   implementation 'org.sensorhub:sensorhub-service-swe:' + oshCoreVersion
   implementation 'org.sensorhub:sensorhub-service-sweapi:' + oshCoreVersion
+  
+  // mp4parser has a transitive dependency on a beta version of slf4j-api, so we explicitly exclude its
+  // dependency and include a non-beta one.
+  implementation('org.mp4parser:isoparser:1.9.56') { exclude group: 'org.slf4j', module: 'slf4j-api' }
+  implementation('org.mp4parser:muxer:1.9.56')     { exclude group: 'org.slf4j', module: 'slf4j-api' }
+  implementation('org.mp4parser:streaming:1.9.56') { exclude group: 'org.slf4j', module: 'slf4j-api' }
+  implementation 'org.slf4j:slf4j-api:1.7.36'
```

Comment:
Do we need to include slf4j here? I think we should get it transitively from sensorhub-core

Reply:
True. (Or at least I should have removed the version number so that the project can inherit the version from elsewhere.) I will remove it.